openassessment/xblock/grade_mixin.py

```diff
@@ -567,15 +567,16 @@ def _assessment_grade_context(self, assessment):
         # If criteria/options in the problem definition do NOT have a "label" field
         # (because they were created before this change),
         # we create a new label that has the same value as "name".
-        for part in assessment['parts']:
-            criterion_label_key = part['criterion']['name']
-            part['criterion']['label'] = criterion_labels.get(criterion_label_key, part['criterion']['name'])
-
-            # We need to be a little bit careful here: some assessment parts
-            # have only written feedback, so they're not associated with any options.
-            # If that's the case, we don't need to add the label field.
-            if part.get('option') is not None:
-                option_label_key = (part['criterion']['name'], part['option']['name'])
-                part['option']['label'] = option_labels.get(option_label_key, part['option']['name'])
+        if assessment is not None:
```

Comment:
[click here](https://github.com/edx/edx-ora2/pull/931/files?w=1#diff-ac25c15240962030fe45023e217279ba) to see this change more clearly without whitespace changes (indented a block)